models/pendo.yml

```
@@ -581,7 +599,7 @@ models:
       - name: _fivetran_id
         description: Fivetran created id hashed on the `param_*` fields.
       - name: page_event_key
-        description: Unique key hashed on ['visitor_id', 'timestamp', 'account_id', 'server_name', 'page_id', 'user_agent', 'remote_ip', '_fivetran_id'].
+        description: Unique key hashed on ['source_relation', 'visitor_id', 'timestamp', 'account_id', 'server_name', 'page_id', 'user_agent', 'remote_ip', '_fivetran_id', 'source_relation'].
```

Assuming `source_relation` is not meant to be included twice.
```suggestion
        description: Unique key hashed on ['source_relation', 'visitor_id', 'timestamp', 'account_id', 'server_name', 'page_id', 'user_agent', 'remote_ip', '_fivetran_id'].
```